.github/workflows/desktop-build.yml

```diff
@@ -230,10 +230,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - target: 13
+          - target: 14
             soc: Intel
```

Comment:
To my understanding, the resulting build will be targeting ARM as that's the base image.
The `soc` value is only used to add a label to the resulting build to differentiale Intel/ARM as we had both before.
We can potentially get rid off it completely if we are now ARM-only for Mac builds, and just put a note in the release notes that we no longer support macOS 13, as well as Intel-based Mac's.

<br>

@fwcd mentioned in https://github.com/Cockatrice/Cockatrice/pull/5033#issuecomment-2216106032 some time ago that using the newer images you can still target older macOS versions. We never looked into it.

Not sure the same is possible for arch.